bundle-workflow/src/run_integ_test.py

```diff
@@ -72,11 +73,11 @@ def main():
 
         if passed_components:
             for component, result in passed_components.items():
-                logging.info(f'PASS: Integration Test for {component} {result[1]} with status code {result[0]}')
+                logging.info(f'PASS: Integration Test for {component[0]} {component[1]} with status code {result}')
 
         if failed_components:
             for component, result in failed_components.items():
-                logging.error(f'FAIL: Integration Test for {component} {result[1]} with status code {result[0]}')
+                logging.error(f'FAIL: Integration Test for {component[0]} {component[1]} with status code {result}')
```

Comment:
Note another problem here that we display results out of order of execution.

Reply:
This can be resolve if we use `OrderedDict` instead of `dict`. I can create an issue for the ordering

Reply:
It should be resolved with a test results class that collects results in a single collection and not two.

Reply:
Added the details in #579

Reply:
+1 to `TestResults` class. It'll make this much cleaner and extensible.

* generateSummaryReport()
* appendResult(component, config, result)

Reply:
the parsing logic will be replaced by generateSummary method